Lib/unittest/mock.py

```diff
@@ -1469,13 +1478,31 @@ def get_original(self):
             )
         return original, local
 
+    @property
+    def is_started(self):
```

Comment:
I guess so... I have committed property setters just in case. Will write tests if needed when we figure out what to do with `temp_original`: do we preserve it and somehow deprecate or anything else

Reply:
Fixed them. A bit ugly but anyway these setters exist not for an intended usecase but for backwards compatibility only